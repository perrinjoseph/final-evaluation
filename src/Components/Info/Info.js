import React from "react";
import Card from "../Card/Card";
import DeveloperDesign from "../../images/developerdesign.svg";
import ResponsiveDesign from "../../images/responsivedesign.svg";
import Inovations from "../../images/innovativesolutions.svg";
import Passion from "../../images/passion.svg";

function Info() {
  return (
    <div class="summary">
      <div className="summary__row">
        <Card
          body={
            "I aim to put my creativity to the test, designing and building unique, meaningful products for clients or merely for my own interests."
          }
          title={"Development and Design"}
          image={DeveloperDesign}
        />
        <Card
          body={
            "I aim to put my creativity to the test, designing and building unique, meaningful products for clients or merely for my own interests."
          }
          title={"Responsive Layouts"}
          image={ResponsiveDesign}
        />
      </div>
      <div className="summary__row">
        <Card
          body={
            "There are still many problems that exist in today's society, including laziness. Luckily, I hope to combat these issues by innovating, developing easy-to-use programs, solutions, or products."
          }
          title={"Ideas and Solutions"}
          image={Inovations}
        />
        <Card
          body={
            "With my profound interest and commitment to my field of study, my projects rarely go unfinished and my problems are never left unresolved. "
          }
          title={"Passion and Dedication"}
          image={Passion}
        />
      </div>
    </div>
  );
}

export default Info;
