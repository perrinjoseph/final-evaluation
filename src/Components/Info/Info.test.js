import { render } from "@testing-library/react";
import Info from "./Info";
describe("Info", () => {
  it("should match the snapshot", () => {
    const { asFragment } = render(<Info>Snapshot Info</Info>);
    expect(asFragment()).toMatchSnapshot();
  });
});
