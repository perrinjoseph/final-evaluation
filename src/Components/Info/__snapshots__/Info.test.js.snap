// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Info should match the snapshot 1`] = `
<DocumentFragment>
  <div
    class="summary"
  >
    <div
      class="summary__row"
    >
      <div
        class="cardglobal"
      >
        <img
          class="cardglobal__img"
          src="developerdesign.svg"
        />
        <section>
          <h3>
            Development and Design
          </h3>
          <p>
            I aim to put my creativity to the test, designing and building unique, meaningful products for clients or merely for my own interests.
          </p>
        </section>
      </div>
      <div
        class="cardglobal"
      >
        <img
          class="cardglobal__img"
          src="responsivedesign.svg"
        />
        <section>
          <h3>
            Responsive Layouts
          </h3>
          <p>
            I aim to put my creativity to the test, designing and building unique, meaningful products for clients or merely for my own interests.
          </p>
        </section>
      </div>
    </div>
    <div
      class="summary__row"
    >
      <div
        class="cardglobal"
      >
        <img
          class="cardglobal__img"
          src="innovativesolutions.svg"
        />
        <section>
          <h3>
            Ideas and Solutions
          </h3>
          <p>
            There are still many problems that exist in today's society, including laziness. Luckily, I hope to combat these issues by innovating, developing easy-to-use programs, solutions, or products.
          </p>
        </section>
      </div>
      <div
        class="cardglobal"
      >
        <img
          class="cardglobal__img"
          src="passion.svg"
        />
        <section>
          <h3>
            Passion and Dedication
          </h3>
          <p>
            With my profound interest and commitment to my field of study, my projects rarely go unfinished and my problems are never left unresolved. 
          </p>
        </section>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
